zlib.yaml

```diff
@@ -13,6 +13,12 @@ package:
   dependencies:
     runtime:
 
+secfixes:
+  1.2.12-r3:
+    - CVE-2022-37434
+  1.2.11-r4:
```

Comment:
should this instead be the first version of wolfi zlib?  i.e. `1.2.12-r2`

Reply:
@kaniini just checking ^

Reply:
1.2.12-r0